.github/workflows/integration-tests-kubevirt.yaml

```diff
@@ -1,119 +0,0 @@
-name: Integration tests Kubevirt
-on:
-  pull_request:
-    types:
-      - opened
-      - reopened
-      - synchronize
-    branches:
-      - main
-      - stable-*
-
-jobs:
-  splitter:
-    continue-on-error: true
-    env:
-      kubernetes: "./kubernetes"
-      kubevirt: "./kubevirt"
-      py_version: 3.9
-    runs-on: ubuntu-latest
-    outputs:
-      test_targets: ${{ steps.splitter.outputs.kubevirt_targets }}
-    steps:
-      - name: Checkout kubernetes.core repository
-        uses: actions/checkout@v3
-        with:
-          path: ${{ env.kubernetes }}
-          ref: ${{ github.event.pull_request.head.sha }}
-
-      - name: Checkout the kubevirt.core collection
-        uses: actions/checkout@v3
-        with:
-          repository: kubevirt/kubevirt.core
-          path: ${{ env.kubevirt }}
-
-      - name: "Set up Python ${{ env.py_version }}"
-        uses: actions/setup-python@v4
-        with:
-          python-version: "${{ env.py_version }}"
-
-      - name: List targets from kubevirt.core collection
-        id: splitter
-        run: python ${{ env.kubernetes }}/tools/kubevirt_list_targets.py ${{ env.kubevirt }}
```

Comment:
Should we also remove this referenced file or is it used elsewhere in this collection?